content/docs/12-enterprise-version/03.5-helm-chart-install-reference.md

```diff
@@ -291,9 +288,36 @@ You can configure Palette to use image swap to download the required images. Thi
       isEKSCluster: true
   ```
 
+### NATS
+
+Palette uses [NATS](https://nats.io) and gRPC for communication between Palette components. Dual support for NATS and gRPC is available. You can enable the deployment of an additional load balancer for NATS. Host clusters deployed by Palette use the load balancer to communicate with the Palette control plane. This is an advanced configuration option, and it is not required for most deployments. Speak with your support representative before enabling this option.
+
+| **Parameters** | **Description** | **Type** | **Default value** |
+| --- | --- | --- | --- |
+| `nats.enabled`| Specifies whether to enable the deployment of a NATS load balancer. | Boolean | `true` |
+| `nats.internal`| Specifies whether to deploy a load balancer or use the host network. If this value is set to `true`, then the remaining NATS parameters are ignored. | Boolean | `true` |
+| `nats.natsUrl`| The NATS URL. This can be a comma separated list of <dns_name:port> mappings for the NATS load balancer service. For example, "message1.dev.spectrocloud.com:4222,message2.dev.spectrocloud.com:4222". This parameter is mandatory if `nats.internal` is set to `false`. If `nats.internal` is set to `true`, you can leave this parameter empty. | String | `""` |
+| `nats.annotations`| A map of key-value pairs that specifies load balancer annotations for NATS. You can use annotations to change the behavior of the load balancer and the Nginx configuration. This is an advanced setting. We recommend you consult with your assigned support team representative prior to modification. | Object | `{}` |
```

Comment:
```suggestion
| `nats.annotations`| A map of key-value pairs that specifies load balancer annotations for NATS. You can use annotations to change the behavior of the load balancer and the NGINX configuration. This is an advanced setting. We recommend you consult with your assigned support team representative prior to modification. | Object | `{}` |
```

Reply:
You can ignore this suggestion - git seems to be having a problem and won't let me delete it.
![CleanShot 2023-08-24 at 13 39 59](https://github.com/spectrocloud/librarium/assets/117382432/6d5e75d6-b4f7-4a7a-a3f7-c311f2558f9b)